src/renderer/src/scenes/dir-select-scene/DirSelectScene.tsx

```diff
@@ -44,38 +61,75 @@ export default function DirSelectScene() {
 
   return (
     <div class="relative grid h-screen place-items-center p-8" style={{ background: GRADIENT }}>
+      <div
+        class="absolute top-0 left-0 w-full h-[50px] flex items-center justify-end"
+        style={{ "-webkit-app-region": "drag" }}
+      >
+        {os() !== "darwin" && <WindowControls />}
+      </div>
       <div class="flex h-full max-h-[720px] w-full max-w-[860px] flex-col justify-between gap-12 overflow-y-auto rounded-2xl border border-stroke/10 bg-regular-material p-8 shadow-2xl">
         <h1 class="pt-12 text-center text-2xl">Welcome to osu! radio</h1>
 
         <div class="flex flex-col gap-4">
           <div class="flex flex-col gap-1.5">
-            <label class="text-sm font-bold text-text">Your osu! folder</label>
-            <div
-              onClick={selectDir}
-              class="flex items-baseline justify-between rounded-xl border border-white/5 bg-regular-material p-1 pl-4"
-            >
-              {" "}
-              <span>{dir() === "" ? "[No folder selected]" : dir()}</span>{" "}
-              <Button
-                variant="secondary"
-                onClick={(e) => {
-                  e.stopPropagation();
-                  selectDir();
-                }}
-              >
-                Select folder
-              </Button>
+            <label class="text-md font-bold text-text">Select an osu! installation:</label>
+            {dirs().length > 0 ? (
+              dirs().map((dir) => (
+                <InstallationCard
+                  directory={dir}
+                  selectedDir={selectedDir}
+                  setSelectedDir={setSelectedDir}
+                />
+              ))
+            ) : (
+              <p>No directories found</p>
+            )}
+            <div class="flex justify-between">
```

Comment:
Update this block to:

``` tsx
<div class="flex justify-between items-baseline pt-2">
              <Button size="medium" variant="secondary" onClick={selectDir}>
                Select a different installation
              </Button>
              <p class="text-sm text-subtext">You can always change this in settings.</p>
            </div>
```

![{816F3C1B-21C9-47F3-A014-3272F67B845C}](https://github.com/user-attachments/assets/c23e907a-48da-44d0-9f89-599aaf7e1c24)